src/Promitor.Scraper.Tests.Unit/Serialization/MetricsDeclaration/YamlSerializationTests.cs

```diff
@@ -20,6 +20,14 @@ protected void AssertMetricDefinition(MetricDefinition deserializedMetricDefinit
             Assert.Equal(serviceBusMetricDefinition.ResourceType, deserializedMetricDefinition.ResourceType);
         }
 
+        protected void AssertMetricDefinition(MetricDefinition deserializedMetricDefinition, AzureQueueMetricDefinition azureQueueMetricDefinition)
```

Comment:
If you introduce this, please remove reference to Azure Queue and use it in existing tests as well.